docs/conf_sso_test_msscolab.rst

```diff
@@ -42,7 +42,7 @@ Then you can generate files, simply by running,
 Enable USE_SAML2
 ----------------
 
-To enable saml2 based login (identity provider-based login), set `USE_SAML2 = True` in the `mslib/mscolab/conf.py` file of the MSS Colab server.
+To enable SAML2-based login (identity provider-based login), set `USE_SAML2 = True` in the `mslib/mscolab/conf.py` file of the MSS Colab server.
 
```

Comment:
a user can't reach this.

You may want to describe the steps differently.

add to the PYTHONPATH where your mscolab_settings.py is defined
To start the process add USE_SAML2 = False

At the enable step set USE_SAML2 = True